checker/tests/nullness/java8inference/Issue402.java

```diff
@@ -3,13 +3,14 @@
 
 // @below-java8-jdk-skip-test
 
-// This test now seems to fail because of Issue 1006
-// @skip-test
-
 import java.util.Comparator;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
+@SuppressWarnings({
+    "nullness",
+    "keyfor"
+}) // Type argument inference does not infer the correct types.
```

Comment:
On the same line as the end of the annotation.
This would be greppable by running a local version of google-java-format that uses longer lines.w

Can you explain how you process these automatically?  Then we could find a solution that could work for you.  I've just been guessing so far because I don't know the grep pattern you use.  Thanks!